### PR TITLE
TT-10301 TT-9763 Update tyk-sync.md

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

https://deploy-preview-3423--tyk-docs.netlify.app/docs/nightly/tyk-sync/
<br>

### Description

1. Rewrite feature list as tables to make it easier to compare difference between Tyk OSS and Tyk Dashboard
2. Link examples to the feature if available
<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-sync.md [hunk 1]
```diff
@@ -9,29 +9,32 @@ aliases:
   - /advanced-configuration/manage-multiple-environments/tyk-sync/
 ---
 
-Tyk Sync is a command line tool and library to manage and synchronise a Tyk installation with your version control system (VCS).
+Tyk Sync is a command-line tool and Go library for synchronising API definitions and Security Policies from a Git repository or file system into Tyk. It allows for the versioning of Tyk configurations to Git or files, as well as one-way sync from Git or files to Tyk. 
+
+Tyk Sync works with Git and any CI/CD tool to ensure that only versioned changes are loaded into your Tyk Environment.  After an API developer has configured and tested the APIs locally, he can use "tyk-sync dump" to convert the APIs to transportable format. Following GitOps standard practises, the API developer should create a Pull-Request on Git to get peer review and version the final API configurations. To sync configurations from Git to a target Tyk installation, use "tyk-sync sync", "tyk-sync publish" or "tyk-sync update" in the deployment pipeline.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-sync.md]: What is meant by version the final API configurations?

```suggestion
Tyk Sync works with Git and any CI/CD tool to ensure that only versioned changes are loaded into your Tyk Environment.  After an API developer has configured and tested the APIs locally, he can use "tyk-sync dump" to convert the APIs to transportable format. Following GitOps standard practises, the API developer should create a Pull-Request on Git for a peer review and version the final API configurations. To sync configurations from Git to a target Tyk installation, use *tyk-sync sync*, *tyk-sync publish* or *tyk-sync update* in the deployment pipeline.
```

File: tyk-docs/content/tyk-sync.md [hunk 2]
```diff
@@ -9,29 +9,32 @@ aliases:
...
 {{< note success >}}
 **Note**  
 
 Tyk Sync works with APIs and Policies. It does not work with Keys. See [Move Keys between environments]({{< ref "advanced-configuration/manage-multiple-environments/move-keys-between-environments" >}}) for details.
 {{< /note >}}
 
-
 ## Features
 
-- Update APIs and Policies on remote Tyk Dashboards
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-sync.md:23]: Add or remove . to/from all bullet points for consistency

File: tyk-docs/content/tyk-sync.md [hunk 3]
```diff
@@ -9,29 +9,32 @@ aliases:
...
-- Update APIs on remote Tyk Community Edition Gateways
-- Publish APIS/Policies to remote Tyk Dashboards
-- Publish APIs to remote Tyk Community Edition Gateways
-- Synchronise a Tyk Dashboard's APIs and Policies with your VCS (one-way, definitions are written to the Dashboard)
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-sync.md:27]:
```suggestion
- Synchronise Tyk Dashboard's APIs and Policies with your VCS (one-way, definitions are written to the Dashboard)
```

File: tyk-docs/content/tyk-sync.md [hunk 4]
```diff
@@ -9,29 +9,32 @@ aliases:
...
-- Synchronise a Tyk Community Edition Gateway APIs with those stored in a VCS (one-way, definitions are written to the Gateway)
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-sync.md:28]:
```suggestion
- Synchronise Tyk Community Edition Gateway APIs with those stored in a VCS (one-way, definitions are written to the Gateway)
```

File: tyk-docs/content/tyk-sync.md [hunk 5]
```diff
@@ -9,29 +9,32 @@ aliases:
...
-- Dump Policies and APIs in a transportable format from a Dashboard to a directory
-- Support for importing, converting and publishing Swagger/OpenAPI JSON files as Tyk Classic APIs (OpenAPI 2.0 and 3.0 are supported).
-- Support for Tyk Classic and Tyk OAS APIs (see note)
-- Specialised support for Git. But since API and policy definitions can be read directly from
-  the file system, it will integrate with any VCS.
-- Show and import [Tyk examples](https://github.com/TykTechnologies/tyk-examples)
+| Feature                                                                    | Tyk OSS | Tyk Dashboard | Example |
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-sync.md]: Maybe add an introduction section that explains what this table shows

File: tyk-docs/content/tyk-sync.md [hunk 6]
```diff
@@ -9,29 +9,32 @@ aliases:
...
 
-- Update APIs and Policies on remote Tyk Dashboards
-- Update APIs on remote Tyk Community Edition Gateways
-- Publish APIS/Policies to remote Tyk Dashboards
-- Publish APIs to remote Tyk Community Edition Gateways
-- Synchronise a Tyk Dashboard's APIs and Policies with your VCS (one-way, definitions are written to the Dashboard)
-- Synchronise a Tyk Community Edition Gateway APIs with those stored in a VCS (one-way, definitions are written to the Gateway)
-- Dump Policies and APIs in a transportable format from a Dashboard to a directory
-- Support for importing, converting and publishing Swagger/OpenAPI JSON files as Tyk Classic APIs (OpenAPI 2.0 and 3.0 are supported).
-- Support for Tyk Classic and Tyk OAS APIs (see note)
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-sync.md:31]:
```suggestion
- Support for Tyk Classic and Tyk OAS APIs (see note below)
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Many thanks for submitting PR Carol. I have reviewed and suggested minor changes. Once updated and peer review has been done we can merge into parent branch.

Can you update the PR title to include the Jira ticket [TT-10301] and add a label to the PR?

File: tyk-docs/content/tyk-sync.md (hunk 1 above)
Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md]: Thanks, the suggestion is applied. Versioning: I meant version control it in Git. Rewritten for better clarity.

File: tyk-docs/content/tyk-sync.md (hunk 2 above)
Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md:23]: These lines are removed.

File: tyk-docs/content/tyk-sync.md (hunk 3 above)
Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md:27]: This line is removed

File: tyk-docs/content/tyk-sync.md (hunk 4 above)
Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md:28]: This line is removed

File: tyk-docs/content/tyk-sync.md (hunk 5 above)
Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md]: Added: "Tyk Sync works with the Open Source *Tyk Gateway* and *Tyk Dashboard* installation. The table below shows features available for each installation type."

File: tyk-docs/content/tyk-sync.md (hunk 6 above)
Comment (carolwkc) [reply tyk-docs/content/tyk-sync.md:31]: Updated in table

**Review by tbuchaillot — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**